Dockerfile

```diff
@@ -0,0 +1,14 @@
+# Dockerfile
+
+FROM node:18
+
+WORKDIR /app
+
+COPY package*.json /app
+COPY yarn.lock /app
+
+RUN yarn install
+
+COPY . /app
+
+CMD [ "yarn", "start" ]
```

Comment:
notice the line ending. I think we could add `pre-commit` to solve this (windows and unix use different characters for line ending, this would result in endless git diff for all new commits)

(optional)

steps:
1. create `.pre-commit-config.yaml`
```yaml
---
# See https://pre-commit.com for more information
# See https://pre-commit.com/hooks.html for more hooks
repos:
- repo: https://github.com/pre-commit/pre-commit-hooks
  rev: v4.0.1
  hooks:
  - id: trailing-whitespace
  - id: end-of-file-fixer
  - id: check-yaml
  - id: check-added-large-files
  - id: check-case-conflict
  - id: check-executables-have-shebangs
  - id: check-json
  - id: check-toml
  - id: detect-private-key
  - id: fix-byte-order-marker
  - id: mixed-line-ending
  ```
  
  2. Install pre-commit on your machine and run `pre-commit install` on this repo
  3. Run `pre-commit run --files **/*`
  4. You should see stripped "new line" in all files.

Reply:
Great tools!, I use `pre-commit run --all-files` to check all files. Did I do it right?